package.json

```
@@ -15,6 +15,8 @@
     "build-satisfactory-icon-exporter": "npx esbuild --platform=node src/Tools/SatisfactoryIconExporter/Exporter.ts --bundle --sourcemap --outfile=src/Tools/SatisfactoryIconExporter/Exporter.js",
     "export-satisfactory-icons": "npm run build-satisfactory-icon-exporter && node src/Tools/SatisfactoryIconExporter/Exporter.js",
     "build-app": "npx esbuild src/main.ts --bundle --outfile=dist/script.js",
-    "build-app-prod": "npx esbuild src/main.ts --bundle --minify=true --outfile=dist/script.js"
+    "build-app-prod": "npx esbuild src/main.ts --bundle --minify=true --outfile=dist/script.js",
+    "run-app-locally": "docker run -it --rm -d -p 8080:80 -v ./dist/:/usr/share/nginx/html --name sankeyfactory.github.io nginx:latest",
+    "stop-app-locally": "docker stop sankeyfactory.github.io"
```

```suggestion
    "run-app-docker": "docker run -it --rm -d -p 8080:80 -v ./dist/:/usr/share/nginx/html --name sankeyfactory nginx:latest",
    "stop-app-docker": "docker stop sankeyfactory"
```
I think these names will be better.